backend/app/core/config.py

```
@@ -1,7 +1,7 @@
 import secrets
 import warnings
 import os
-from typing import Annotated, Any, Literal
+from typing import Any, Literal
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)